packaging/torcharrow/meta.yaml

```diff
@@ -20,20 +20,22 @@ requirements:
     - typing
     - tabulate
     - pyarrow
-    - arrow
```

Comment:
btw `arrow` on conda is a completely different lib ...

Reply:
right , or `pyarrow` i guess? https://arrow.apache.org/docs/python/install.html

Reply:
and `arrow-cpp`